shift_detector/precalculations/TextMetadata.py

```diff
@@ -427,6 +428,30 @@ def metadata_function(self, text):
         return textstat.text_standard(text, True)
 
 
+class PartOfSpeechMetadata(GenericTextMetadata):
+
+    @staticmethod
+    def metadata_name() -> str:
+        return 'pos_tags'
+
+    def metadata_return_type(self) -> ColumnType:
+        return ColumnType.categorical
+
+    @staticmethod
+    def tag_histogram(text):
+        tokenized_text = nltk.word_tokenize(text)
+        tagged_text = nltk.pos_tag(tokenized_text)
+        simplified_tagged_text = [(word, nltk.map_tag('en-ptb', 'universal', tag)) for word, tag in tagged_text]
+        tagdict = defaultdict(int)
+        for word in simplified_tagged_text:
+            tagdict[word[1]] += 1
+        return tagdict
+
+    def metadata_function(self, text):
+        # works only for english texts.
```

Comment:
this should be reflected in the code then. Better to issue a warning or throw an error if a different language is set. I think the language can be either set globally or may be inferred per row. If either option is chosen it should warn or error